test/randomized_benchmarking/test_randomized_benchmarking.py

```diff
@@ -350,6 +346,22 @@ def test_non_clifford_interleaved_element(self):
             lengths=lengths,
         )
 
+    def test_interleaving_delay(self):
```

Comment:
You should skip this test until the 0.20.1 terra patch is released which includes the required fix there.